samples/nrf9160/modem_shell/boards/thingy91_nrf9160_ns.overlay

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+ / {
+	leds {
+		compatible = "gpio-leds";
+		red_led: led_1 {
```

Comment:
Would it be possible to change the `led0` alias instead, so that `led0` would be the blue led instead of red?

Reply:
You got good point on this, but unfortunately with aliases it couldn't be done. But what I did, I changed LTE connected LED from DK LED#1 to LED#3. That does the wanted functionality with THingy91 and blue LED is on when LTE is connected. Thus, removed this overlay